src/core/search/search_parser_test.cc

```diff
@@ -12,9 +12,9 @@ namespace dfly::search {
 
 using namespace std;
 
-class SearchParserTest : public ::testing::Test {
+class SearchTest : public ::testing::Test {
```

Comment:
I get why you changed the test names in `search_test.cc`, but did you intend to rename here as well?

Reply:
Ops, I didn't, vscode did its job to well